src/dmd/backend/cdef.d

```diff
@@ -22,11 +22,36 @@ extern (C++):
 @nogc:
 nothrow:
 
-//struct Classsym;
-//struct Symbol;
-//struct LIST;
 struct param_t;
 
+enum VERSION = "9.00.0";        // for banner and imbedding in .OBJ file
+enum VERSIONHEX = "0x900";      // for __DMC__ macro
+enum VERSIONINT = 0x900;        // for precompiled headers and DLL version
+
+version (SCPP)
+    version = XVERSION;
+version (SPP)
+    version = XVERSION;
+version (HTOD)
+    version = XVERSION;
+
+version (XVERSION)
+{
+    extern (D) template xversion(string s)
+    {
+        enum xversion = mixin(`{ version (` ~ s ~ `) return true; else return false; }`)();
+    }
+
+    enum IN_GCC     = xversion!`IN_GCC`;
+
+    enum TARGET_LINUX   = xversion!`linux`;
+    enum TARGET_OSX     = xversion!`OSX`;
+    enum TARGET_FREEBSD = xversion!`FreeBSD`;
+    enum TARGET_OPENBSD = xversion!`OpenBSD`;
+    enum TARGET_SOLARIS = xversion!`Solaris`;
+    enum TARGET_WINDOS  = xversion!`Windows`;
```

Comment:
As DragonFlyBSD Support was merged yesterday, shouldn't it also be specified here ?
```
enum TARGET_DRAGONFLYBSD = xversion!`DragonFly`;
```

Reply:
done - see next line